tests/onnxruntime/test_optimization.py

```diff
@@ -171,6 +170,7 @@ def test_compare_original_seq2seq_model_with_optimized_model(self, model_cls, mo
 
     # Contribution note: Please add test models in alphabetical order. Find test models here: https://huggingface.co/hf-internal-testing.
     SUPPORTED_IMAGE_ARCHITECTURES_WITH_MODEL_ID = (
+        (ORTModelForImageClassification, "hf-internal-testing/tiny-random-segformer"),
```

Comment:
there's no `tiny-random-segformer` on the hub, there's however `hf-internal-testing/tiny-random-SegformerForImageClassification`.

Reply:
I am able to use both the processor and the model for `"hf-internal-testing/tiny-random-segformer"`:

```python
import torch
from transformers import AutoModel, AutoImageProcessor
model_name = "hf-internal-testing/tiny-random-segformer"
processor = AutoImageProcessor.from_pretrained(model_name)
model = AutoModel.from_pretrained(model_name)
dummy_image = torch.rand(1, 3, 224, 224)
inputs = processor(images=dummy_image, return_tensors="pt")
with torch.no_grad():
    outputs = model(**inputs)
print(outputs)
```

Maybe I'm misunderstanding something?  But the model seems to work.

Reply:
There's a page for the model too: https://huggingface.co/hf-internal-testing/tiny-random-segformer

Reply:
okay weird I couldn't find it for some reason.